web/src/components/toggle.js

```diff
@@ -73,7 +73,7 @@ const InfoButton = styled.div`
   }
 `;
 
-const Toggle = ({
+export default ({
   className,
```

Comment:
Not sure if I made myself clear enough, but in https://github.com/tmrowco/electricitymap-contrib/pull/2350#discussion_r402182450, I was suggesting removing this prop if it's not being used anymore.

Reply:
ah got it! Will do